modules/sdp/SDP.js

```diff
@@ -95,8 +95,18 @@ export default class SDP {
             updatedMidIndices.push(idx);
 
             if (isAdd) {
+                let updatedMsid = msid;
+
+                // If the msid is not in the <stream ID> <track ID> format, create msid in that format.
+                // Chrome's older versions (upto 117) expect the msid to be in this format.
+                if (msid.split(' ').length !== 2
+                    && browser.isChromiumBased()
```

Comment:
Why the browser check? Doesn't it make sense to do it always?

Reply:
As a precaution. The newer versions go into a renegotiation loop when I use the same string for both streamId and trackId and that was the reason I didn't implement this fix the first time around.